Formula/dracula-theme.rb

```diff
@@ -0,0 +1,26 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+class DraculaTheme < EmacsFormula
+  desc "A dark color theme available for a number of editors."
+  homepage "https://github.com/dracula/emacs"
+  url "https://github.com/dracula/emacs/archive/v1.2.5.tar.gz"
+  sha256 "bc78039c90d5d27f6b1ccb7b32fabb3ecbf0d485421bc63ae5d5d6836b9780b4"
+  head "https://github.com/dracula/emacs.git"
+
+  bottle :unneed
+
+  depends_on :emacs => "24.1"
+
+  def install
+    elisp.install "dracula-theme.el", "dracula-theme.elc"
```

Comment:
You'll need `byte_compile "dracula-theme.el"` above this line  if you want the `.elc`.